### ISSUE TITLE
Improve "swapped" page

### ISSUE THREAD

**Issue by thomaseizinger** [comit-network/droplet#100]
After a successful swap, we redirect the user to a "swapped" page where they can click on a link to the blockexplorer.

Other than that, this page does not display anything useful.

We should improve that by saying something like: "Your swap is complete" etc.

**Comment by bonomat** [comment]
Agree, I didn't care much about this screen when doing the initial implementation. If anyone wants to pick it up, our mocks had an idea how to design this: 

Transaction confirmation screen: 
https://www.figma.com/proto/MxcADlWgz6Z8XdwXwFuzVU/BTC%2FUSDT?node-id=2%3A3940&scaling=scale-down


Transaction confirmed screen: 
https://www.figma.com/proto/MxcADlWgz6Z8XdwXwFuzVU/BTC%2FUSDT?node-id=2%3A4328&scaling=scale-down
